src/vm/gchelpers.cpp

```diff
@@ -1319,6 +1341,13 @@ void ErectWriteBarrierForMT(MethodTable **dst, MethodTable *ref)
     
     if (ref->Collectible())
     {
+#ifdef FEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP
+        if (SoftwareWriteWatch::IsEnabledForGCHeap())
+        {
```

Comment:
My understanding is that this only applies to objects from collectible assemblies, and doesn't apply to CoreCLR. I don't know what the significance of the code here is, but I just added the update here for completeness, as it looks like a place where the write watch table would need to be updated if the feature applies.